src/content/en/2019/page-weight.md

```diff
@@ -6,809 +6,765 @@ description: Page Weight chapter of the 2019 Web Almanac covering why page weigh
+      <td>318</td>
+      <td>6</td>
+      <td>65</td>
+      <td>5</td>
+      <td>59</td>
+      <td>4</td>
+    </tr>
+  </table>
+  <figcaption>Figure 3. Page weight on mobile broken down by resource type</figcaption>
+</figure>
 
+#### Desktop
 
-# Analysis
+<figure>
+  <table>
+    <tr>
+      <th>Percentile</th>
+      <th>Total (KB)</th>
+      <th>HTML (KB)</th>
+      <th>JS (KB)</th>
+      <th>CSS (KB)</th>
+      <th>Image (KB)</th>
+      <th>Document (KB)</th>
+    </tr>
+    <tr>
+      <td>90</td>
+      <td>6945</td>
+      <td>110</td>
+      <td>1131</td>
+      <td>240</td>
+      <td>5220</td>
+      <td>52</td>
+    </tr>
+    <tr>
+      <td>75</td>
+      <td>3774</td>
+      <td>58</td>
+      <td>721</td>
+      <td>129</td>
+      <td>2434</td>
+      <td>26</td>
+    </tr>
+    <tr>
+      <td>50</td>
+      <td>1934</td>
+      <td>27</td>
+      <td>391</td>
+      <td>62</td>
+      <td>983</td>
+      <td>14</td>
+    </tr>
+    <tr>
+      <td>25</td>
+      <td>924</td>
+      <td>12</td>
+      <td>186</td>
+      <td>26</td>
+      <td>319</td>
+      <td>8</td>
+    </tr>
+    <tr>
+      <td>10</td>
+      <td>397</td>
+      <td>6</td>
+      <td>76</td>
+      <td>8</td>
+      <td>78</td>
+      <td>4</td>
+    </tr>
+  </table>
+  <figcaption>Figure 4. Page weight on desktop broken down by resource type</figcaption>
+</figure>
+
+### Page Weight Over Time
+
+Over the past year the median size of a desktop site increased by 434KB, and the median size of a mobile site increased by 179KB. Images are overwhelmingly driving this increase.
 
-Note: The statistics in this section are all based on the _transfer size_ of a page and its resources. Not all resources on the web are compressed before sending, but if they are, this analysis uses the compressed size.
+#### Mobile
 
+<figure>
+  <table>
+    <tr>
+      <th>Percentile</th>
+      <th>Total (KB)</th>
+      <th>HTML (KB)</th>
+      <th>JS (KB)</th>
+      <th>CSS (KB)</th>
+      <th>Image (KB)</th>
+      <th>Document (KB)</th>
+    </tr>
+    <tr>
+      <td>90</td>
+      <td>+376</td>
+      <td>-50</td>
+      <td>+46</td>
+      <td>+36</td>
+      <td>+648</td>
+      <td>+2</td>
+    </tr>
+    <tr>
+      <td>75</td>
+      <td>+304</td>
+      <td>-7</td>
+      <td>+34</td>
+      <td>+21</td>
+      <td>+281</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <td>50</td>
+      <td>+179</td>
+      <td>-1</td>
+      <td>+27</td>
+      <td>+10</td>
+      <td>+106</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <td>25</td>
+      <td>+110</td>
+      <td>-1</td>
+      <td>+16</td>
+      <td>+5</td>
+      <td>+36</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <td>10</td>
+      <td>+72</td>
+      <td>0</td>
+      <td>+13</td>
+      <td>+2</td>
+      <td>+20</td>
+      <td>+1</td>
+    </tr>
+  </table>
+  <figcaption>Figure 5. Change (KB) in page size vs. 2018</figcaption>
+</figure>
 
-## Page Weight
+#### Desktop
```

Comment:
WDYT about combining these tables so that desktop and mobile alternate? With the default table styling of highlighting alternate rows, it might be a convenient way to compare the two. Having the tables separate would make comparison more difficult.